insightedge-integration-tests/src/test/scala/org/insightedge/spark/cluster/ClusterInstallationSpec.scala

```diff
@@ -35,14 +35,15 @@ class ClusterInstallationSpec extends FlatSpec with BeforeAndAfter {
   val scriptsDir = getClass.getClassLoader.getResource("docker/cluster-install/").getFile
 
   "insightedge.sh" should "install cluster" in {
-    val packagerDir = findPackagerDir(new File(".")).getOrElse(fail(s"Cannot find $PackagerDirName directory"))
 
-    println(s"Package dir: $packagerDir")
-    val zipDir = s"$packagerDir/target/$BuildEdition"
+    var zipDir = Option(System.getProperty("dist.dir")).getOrElse("")
```

Comment:
Please prefer don't use `var`. This can be refactored with something like
val zipDir = Option(System.getProperty("dist.dir")).getOrElse{
  val packagerDir = findPackagerDir(new File(".")).getOrElse(fail(s"Cannot find $PackagerDirName directory"))
 println(s"Package dir: $packagerDir")
 s"$packagerDir/target/$BuildEdition"
}